TalkAloud/TalkAloud/AudioEngine.swift

```diff
@@ -9,7 +9,12 @@
 import Foundation
 import AVFoundation
 
+protocol AudioEngineStateChangeDelegate: class {
+    func didUpdateAudioState(willSetAudioEngineStateTo audioState: AudioEngineState)
```

Comment:
You can rename to `func didUpdateAudioState(with audioState: AudioEngineState)`

Reply:
Yeah, I thought of that when I pushed up my code haha.